components/consents/api/pom.xml

```diff
@@ -34,6 +34,10 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
```

Comment:
according to https://github.com/findbugsproject/findbugs `FindBugs is now SpotBugs` - since we are updating dependencies should we try to move to `SpotBugs` instead?

Reply:
There is no newer version of this jar under the spotbugs name, so there is nothing to switch to.